src/gnu/ACG.cpp

```diff
@@ -20,7 +20,7 @@ Foundation, 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA.
 #ifdef __GNUG__
 #pragma implementation
 #endif
-#include <ACG.h>
+#include <gnu/ACG.h>
```

Comment:
[SF.12: Prefer the quoted form of #include for files relative to the including file and the angle bracket form everywhere else](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#sf12-prefer-the-quoted-form-of-include-for-files-relative-to-the-including-file-and-the-angle-bracket-form-everywhere-else) suggests this should be `#include "ACG.h"`, right?

Reply:
right. My process was automatic ^^ I can fix it. (btw it will be superseeded by #2391